src/script/util/ClipboardUtil.js

```diff
@@ -17,10 +17,7 @@
  *
  */
 
-window.z = window.z || {};
-window.z.util = z.util || {};
-
-z.util.ClipboardUtil = {
+const ClipboardUtil = {
```

Comment:
You could even export individual named functions here (instead of an object as the default export).
We would benefit from tree shaking with this method (BTW it seems the `pasteText` method is never used, if not, it can be removed)